src/main/java/se/sundsvall/messaging/api/model/request/EmailRequest.java

```diff
@@ -41,7 +43,9 @@ public record EmailRequest(
 
 	@ArraySchema(schema = @Schema(implementation = Attachment.class)) List<@Valid Attachment> attachments,
 
-	@Schema(description = "Headers") Map<Header, List<@Pattern(regexp = "^<.{1,1000}@.{1,1000}>$", message = "Header values must start with '<', contain '@' and end with '>'") String>> headers) {
+	@Schema(description = "Headers") Map<Header, List<@Pattern(regexp = "^<.{1,1000}@.{1,1000}>$", message = "Header values must start with '<', contain '@' and end with '>'") String>> headers,
+
+	@Schema(description = "Municipality Id", hidden = true) String municipalityId) {
```

Comment:
@JsonIgnore

Reply:
Like: `@Schema(description = "Municipality Id", hidden = true) @JsonIgnore String municipalityId`

Goes for all them requeties